storage/innobase/xtrabackup/test/t/skip_orphan_table.sh

```diff
@@ -0,0 +1,83 @@
+. inc/common.sh
+
+remote_dir=$TEST_VAR_ROOT/var1/remote_dir/subdir
+undo_dir=$TEST_VAR_ROOT/var1/undo_dir/subdir
+mkdir -p $remote_dir
+mkdir -p $undo_dir
+
+start_server --innodb_file_per_table --innodb_directories="$remote_dir;$undo_dir"
+
+$MYSQL $MYSQL_ARGS test <<EOF
+CREATE TABLESPACE TS1 ADD DATAFILE '$remote_dir/ts1.ibd';
+CREATE TABLESPACE TS2 ADD DATAFILE '$remote_dir/ts2_name_mismatch.ibd';
+CREATE TABLE t(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
+ DATA DIRECTORY = '$remote_dir' ENGINE=InnoDB;
+INSERT INTO t(c) VALUES (1), (2), (3), (4), (5), (6), (7), (8);
+CREATE TABLE b(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
+  TABLESPACE TS1  ENGINE=InnoDB;
+INSERT INTO b(c) VALUES (1), (2), (3), (4), (5), (6), (7), (8);
+CREATE TABLE c(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
```

Comment:
can you please create a table with a space in the name '  c' ?

Reply:
Did not understand _the space in the name_. like c_TS1?

Reply:
tablespace name contains spaces in middle 'a  b     c' like this.

Reply:
fixed